src/controllers/busControllers.js

```diff
@@ -0,0 +1,124 @@
+import { paginate } from 'paginate-info';
+import successRes from '../helpers/successHandler';
+import errorRes from '../helpers/errorHandler';
+import model from '../database/models';
+
+const { Bus } = model;
+export const createBus = async (req, res) => {
+  try {
+    const existingBus = await Bus.findOne({
+      where: { plate: req.body.plate },
```

Comment:
can we have a variable that stores the `plateNo` and call it here instead of having` req.body.plateNo` ? and let's `plate` to `plateNo`